ui.apps/src/main/content/jcr_root/apps/core/fd/components/form/textinput/v1/textinput/clientlibs/site/css/textinputview.css

```diff
@@ -27,17 +27,60 @@
 }
 
 .cmp-adaptiveform-textinput__longdescription {
-
+    font-size: .875rem;
+    padding-left: 1rem;
+    padding-bottom: 0.125rem;
+    padding-top: 0.125rem;
+    padding-right: 1rem;
```

Comment:
no css as part of the component. It should come separately. 

cc @rismehta @nit23uec